libimage/image.go

```diff
@@ -256,6 +256,10 @@ func (i *Image) TopLayer() string {
 	return i.storageImage.TopLayer
 }
 
+func (i *Image) mappedTopLayers() []string {
```

Comment:
(I’m not convinced that this function adds value, but now that it exists, I’m fine with leaving it in.)